### PR TITLE
add validation on appointment_made_date, on opd_check_in form

### PR DESCRIPTION
**Pull request by roywanyaga**
Closes issue #465

### CODE REVIEW

**Comment by roywanyaga** [comment]
Tests failed. There is a dependency with bintray. Fixing that.

File: build.gradle [hunk 1]
```diff
@@ -25,7 +25,6 @@ allprojects {
         maven { url "https://repo.maven.apache.org/maven2" }
         maven { url "http://cloudant.github.io/cloudant-sync-eap/repository" }
         maven { url "https://s3.amazonaws.com/repo.commonsware.com" }
-        maven { url 'https://dl.bintray.com/ibm-watson-health/ibm-fhir-server-releases'}
```
Comment (hamza-vd) [inline build.gradle:28]: Have you confirmed this fix is correct? I mean it might not cause issues with other libraries?

Comment (roywanyaga) [reply build.gradle:28]: @hamza-vd yes. Allan helped to confirm this. "Hi Roy. After the sunset of Bintray  services, we no longer have dependencies resolving from Bintray. That line should be deleted from where the repositories are declared. The IBM-FHIR dependency should resolve from jcenter()"

**Review by hamza-vd — APPROVED** [review]
LGTM

**Review by dubdabasoduba — APPROVED** [review]

**Pull request merged**